gemrb/plugins/CHUImporter/CHUImporter.cpp

```diff
@@ -377,7 +377,7 @@ Window* CHUImporter::GetWindow(ScriptingId wid) const
 				TextArea* ta = new TextArea( ctrlFrame, fnt, ini, fore, init, back );
 
 				if (SBID != 0xffff) {
-					ScrollBar* sb = GetControl<ScrollBar>(SBID, win);
+					ScrollBar* sb = dynamic_cast<ScrollBar*>(GetControl(SBID, win));
```

Comment:
Technically, they are equivalent. But since `GetControl<T>` originates from `core/GUI` or so, the linking process fails as clang refuses to link a templated function here. Maybe there are hacks around that but for my eyes, it makes sense: How should `libgemrb_core` be layouted internally to work with arbitrary extern `T`'d-calls?